core/src/test/scala/com/salesforce/op/filters/FeatureDistributionTest.scala

```diff
@@ -238,4 +241,21 @@ class FeatureDistributionTest extends FlatSpec with PassengerSparkFixtureTest wi
     intercept[IllegalArgumentException](fd1.jsDivergence(fd1.copy(name = "boo"))) should have message
       "requirement failed: Name must match to compare or combine feature distributions: A != boo"
   }
+
+  it should "not serialize cardEstimate field" in {
+    val fd1 = FeatureDistribution("A", None, 10, 1, Array(1, 4, 0, 0, 6),
+      Array.empty, Some(Moments(1.0)), Some(TextStats(Map("foo" -> 1, "bar" ->2))),
+      FeatureDistributionType.Scoring)
+    FeatureDistribution.toJson(Seq(fd1)) shouldNot include ("cardEstimate")
+
+    val json =
+      """[{"name":"A","count":10,"nulls":1,"distribution":[1.0,4.0,0.0,0.0,6.0],
+        |"summaryInfo":[],"moments":{"m0":1,"m1":1.0,"m2":0.0,"m3":0.0,"m4":0.0},
+        |"cardEstimate":{"valueCounts":{"foo":1,"bar":2}},"type":"Scoring"}]
+        |""".stripMargin
+    FeatureDistribution.fromJson(json) match {
```

Comment:
more concisely:
```
FeatureDistribution.fromJson(json) shouldBe Success(Seq(fd1))
```

Reply:
I think we need the test case for being able to deserialize where cardEstimate is present in JSON

Reply:
yes, this test case is doing so

Reply:
sorry, I meant the complementary case then without cardEstimate in JSON